sa/sa.go

```diff
@@ -1065,32 +1069,38 @@ func (ssa *SQLStorageAuthority) getFQDNSetsBySerials(serials []string) ([][]byte
 		qmarks[i] = "?"
 	}
 	query := "SELECT setHash FROM fqdnSets " +
-		"WHERE serial IN (" + strings.Join(qmarks, ",") + ") "
+		"WHERE serial IN (" + strings.Join(qmarks, ",") + ")"
 	_, err := ssa.dbMap.Select(
 		&fqdnSets,
 		query,
 		params...)
 
-	if err != nil && err != sql.ErrNoRows {
+	// NOTE(@cpu): We don't explicitly check if `err != sql.ErrNoRows` here
```

Comment:
There it is: Fixed in https://github.com/letsencrypt/boulder/pull/2832/commits/76e30d1e09a0d5c751d60a3efd2124abbde19ab6